compose/services.yml

```diff
@@ -79,17 +79,17 @@ services:
       REGISTRY2_CACHE_DB: 0
       REGISTRY2_CACHE_MAXMEMORY_MB: 1024 # megabytes
       REGISTRY2_CACHE_MAXMEMORY_POLICY: allkeys-lru
-      REGISTRY2_S3_BUCKET:
-      REGISTRY2_S3_KEY:
-      REGISTRY2_S3_SECRET:
+      REGISTRY2_S3_BUCKET: ${OPENBALENA_S3_BUCKET}
+      REGISTRY2_S3_KEY: ${OPENBALENA_S3_ACCESS_KEY}
+      REGISTRY2_S3_SECRET: ${OPENBALENA_S3_SECRET_KEY}
```

Comment:
You also need to specify `REGISTRY2_S3_REGION_ENDPOINT`, given this is not targeting Amazon's S3.